curriculum/LICENSE.md

```diff
@@ -424,4 +424,4 @@ understandings, or agreements concerning use of licensed material. For
 the avoidance of doubt, this paragraph does not form part of the
 public licenses.
 
-Creative Commons may be contacted at creativecommons.org.
+Creative Commons may be contacted at [creativecommons.org.](creativecommons.org "Creative Commons")
```

Comment:
markdown syntax is wrong and leads to a 404 page. I think the quoted words need removing in the round brackets

Reply:
I am not sure where you are wanting the "Creative Commons" part to be, but if you are wanting it to appear to the write of `creativecommons.org.`, then "Creative Commons:" should be inside the `[ ]` and not the `( )`.  Also, you need a fully-formed url inside the `( )`.

[creativecommons.org "Create Commons"](https://creativecommons.org)

Reply:
@ceejayoz An accident by me.